pontoon/base/models.py

```diff
@@ -2245,6 +2246,12 @@ def reset_active_translations(self, locale):
 
         translations.filter(pk__in=unreviewed_pks).update(active=True)
 
+    def get_or_create(self, defaults=None, **kwargs):
```

Comment:
I'm not sure if you need to override this method. Do you have a use-case for this?

Reply:
I'll guess we'll add logic to calculate and store word count in a followup PR?